### PR TITLE
Adding ORDER_INFO, FILLS to huobi spot

### PR DESCRIPTION
**Pull request by Acid3croco**
### Description of code - what bug does this fix / what feature does this add?

- [X] - Tested on live api posting limit/market orders
- [X] - Changelog updated
- [  ] - Tests run and pass
Error with type import, I may be missing something to execute them?
```ModuleNotFoundError: No module named 'cryptofeed.types'```

- [X] - Flake8 run and all errors/warnings resolved

### CODE REVIEW

**Comment by bmoscon** [comment]
@Acid3croco - if you have downloaded the source code to develop from, you need to run `python setup.py develop`. This will build the Cython data types into that file - `cryptofeed.types`. I'll take a look at what you have here, but I won't merge anything that is un-tested, so let me know when you're able to get that working and tested

File: cryptofeed/exchanges/huobi.py [hunk 1]
```diff
@@ -14,27 +20,41 @@
...
 
 LOG = logging.getLogger('feedhandler')
 
 
 class Huobi(Feed):
     id = HUOBI
-    websocket_endpoints = [WebsocketEndpoint('wss://api.huobi.pro/ws')]
-    rest_endpoints = [RestEndpoint('https://api.huobi.pro', routes=Routes('/v1/common/symbols'))]
+    is_auth = False
+    rest_endpoints = [
```
Comment (bmoscon) [inline cryptofeed/exchanges/huobi.py:33]: please undo this change - looks like the line did not change, but you used some tool that made a 1 line declaration into a 3 line.

File: cryptofeed/exchanges/huobi.py [hunk 2]
```diff
@@ -48,7 +68,8 @@ def _parse_symbol_data(cls, data: dict) -> Tuple[Dict, Dict]:
         for e in data['data']:
             if e['state'] == 'offline':
                 continue
-            base, quote = e['base-currency'].upper(), e['quote-currency'].upper()
+            base, quote = e['base-currency'].upper(
```
Comment (bmoscon) [inline cryptofeed/exchanges/huobi.py:71]: undo this formatting change

File: cryptofeed/exchanges/huobi.py [hunk 3]
```diff
@@ -62,10 +83,18 @@ async def _book(self, msg: dict, timestamp: float):
         pair = self.exchange_symbol_to_std_symbol(msg['ch'].split('.')[1])
         data = msg['tick']
         if pair not in self._l2_book:
-            self._l2_book[pair] = OrderBook(self.id, pair, max_depth=self.max_depth)
```
Comment (bmoscon) [inline cryptofeed/exchanges/huobi.py:65]: undo this

File: cryptofeed/exchanges/huobi.py [hunk 4]
```diff
@@ -62,10 +83,18 @@ async def _book(self, msg: dict, timestamp: float):
...
+            self._l2_book[pair] = OrderBook(self.id,
+                                            pair,
+                                            max_depth=self.max_depth)
 
-        self._l2_book[pair].book.bids = {Decimal(price): Decimal(amount) for price, amount in data['bids']}
-        self._l2_book[pair].book.asks = {Decimal(price): Decimal(amount) for price, amount in data['asks']}
+        self._l2_book[pair].book.bids = {
```
Comment (bmoscon) [inline cryptofeed/exchanges/huobi.py:90]: undo this

**Review by bmoscon — CHANGES_REQUESTED** [review]

**Comment by bmoscon** [comment]
@Acid3croco looks like a lot of changes that are just formatting changes. Please undo those and I'll take another look. Have a look at the contributing doc - https://github.com/bmoscon/cryptofeed/blob/master/CONTRIBUTING.md - under PRs it specifically calls out formatting changes like this.

File: cryptofeed/exchanges/huobi.py [hunk 5]
```diff
@@ -155,36 +180,196 @@ async def _candles(self, msg: dict, symbol: str, interval: str, timestamp: float
...
+            LOG.debug(f"{conn.uuid}: Authenticating with message: {auth}")
+            await conn.write(json.dumps(auth))
+
+    async def message_handler(self, msg: str, conn: AsyncConnection, timestamp: float):
+        if not any(self.is_authenticated_channel(self.exchange_channel_to_std(chan)) for chan in conn.subscription):
+            # unzip message if its from a compressed channel
+            msg = zlib.decompress(msg, 16 + zlib.MAX_WBITS)
         msg = json.loads(msg, parse_float=Decimal)
 
+        LOG.debug(f"{conn.uuid}: Received message: {msg}")
```
Comment (bmoscon) [inline cryptofeed/exchanges/huobi.py:353]: even in a debug scenario I dont think we want to log every message, that would be a massive amount of volume.

**Review by bmoscon — CHANGES_REQUESTED** [review]

**Comment by Acid3croco** [comment]
Hello, ima fix everything and will come back with something cleaner, thanks for the feedback

**Comment by xiva-wgt** [comment]
- Tests run and pass
Error with type import, I may be missing something to execute them?
ModuleNotFoundError: No module named 'cryptofeed.types'

Before code import cryptofeed need add this code
import pyximport
pyximport.install(language_level=3)

**Comment by bmoscon** [comment]
@xiva-wgt if you download the source code you need to run setup.py to build the cython modules

**Comment by bmoscon** [comment]
@Acid3croco if you ever get back to this please open a new PR or re-open this one. Closing since this seems to be abandoned.

**Pull request closed without merging**